api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -294,6 +295,7 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
   @Override
   public ResponseEntity<EmptyResponse> deleteWorkspace(String workspaceNamespace,
       String workspaceId) {
+    // TODO: do we want to delete any workspace resources?
```

Comment:
currently we are not; we need to save workbench metadata and collaborator lists (the `user_workspace` table), but could get rid of other pieces.  however if we keep that stuff we can easily implement workspace restore